src/pages/Booking/MusicalBooking/MusicalItem.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import { useBookingStore } from '../store/store';
+
+const Item = styled.div`
+  user-select: none;
+  padding: 16px;
+  display: flex;
+  flex-shrink: 0;
+  width: 100%;
+  background: #ffffff;
+  transition: 0.3s;
+  cursor: pointer;
+
+  ${props =>
+    props.isSelected === true &&
+    css`
+      background: ${({ theme }) => theme.colors.primary.main};
+      color: #ffffff;
+    `}
+
+  &:hover {
+    ${props => props.isClicked === false && css``}
+  }
+`;
+
+const MusicalThumb = styled.div`
+  ${props =>
+    props.imageUrl &&
+    css`
+      background-image: url(${props.imageUrl});
+    `}
+  transition: .5s;
+  flex-shrink: 0;
+  background-size: cover;
+  height: 120px;
+  width: 90px;
+  border-radius: 5px;
+`;
+const MusicalTextContent = styled.div`
+  margin-left: 16px;
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  justify-content: center;
+`;
+
+const Header = styled.div`
+  display: flex;
+  width: 100%;
+  align-items: center;
+
+  p {
+    font-weight: bold;
+    padding-left: 10px;
+    line-height: 0;
+  }
+`;
+
+const AgeLimit = styled.div`
+  background-color: ${props =>
+    props.ageLimit === 18
+      ? '#FF0363'
+      : props.ageLimit === 15
+      ? '#F48131'
+      : props.ageLimit === 12
+      ? '#F5C951'
+      : '#36C640'};
```

Comment:
```suggestion
  background-color: ${({ageLimit}) =>{
  const obj = {
  18:'#FF0363',
  15:'#F48131',
  12:'#F5C951'
  }
  return obj[ageLimit]||'#36C640'}
```
조건문으로 props로 들어오는 ageLimit을 다 비교해서 값을 반활할 수도 있는데,
if 문은 객체로 표현할 수 있습니다. 
객체는 key가 들어오면 바로 값을 반환해서 직관적이고 편리합니다.
|| 연산자의 부가기능인 단축평가로, 좌항이 null 혹은 undefined일 경우 우항을 반환하는 기능을합니다.
18,15,12가 아닌경우에는 '#36C640' 이 값을 반환하도록 하는 문법 입니다.